cpp/src/io/orc/dict_enc.cu

```diff
@@ -72,6 +72,10 @@ static __device__ void LoadNonNullIndices(volatile dictinit_state_s* s,
                                           Storage& temp_storage)
 {
   if (t == 0) { s->nnz = 0; }
+  if (s->chunk.num_rows <= 0) {
```

Comment:
copyrights need to be updated

Reply:
@nvdbaranec Is there anything similar in parquet code?